app/core/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/progress/HedvigFullScreenCenterAlignedProgress.kt

```diff
@@ -29,6 +37,25 @@ fun HedvigFullScreenCenterAlignedProgress(
   }
 }
 
+@Composable
+fun HedvigFullScreenCenterAlignedProgressDebounced(
+  modifier: Modifier = Modifier,
+  show: Boolean = true,
+  debounceMillis: Long = DEBOUNCE_MILLIS,
+) {
+  var debounce by remember { mutableStateOf(false) }
```

Comment:
ok now I understood what this was for, initially I thought the debounce was for the opposite effect, that it should still show for X milliseconds even if show turns false.
So I understood the flipped thing of what this actually does. So this looks good yeah, now that I understood it. Sorry 😅